### PR TITLE
Qual 73 add das configuration

### PR DESCRIPTION
**Pull request by DannyGB**
Adds the standard das configuration infrastructure to the project
To get this to work you will also need to add configuration to the local storage emulator

### CODE REVIEW

**Review by MarieDale3 — APPROVED** [review]

File: src/SFA-DAS-HMRC-API-Stub/Data/Configuration/Declaration.cs [hunk 1]
```diff
@@ -11,12 +11,12 @@ public class Declaration : IEntityTypeConfiguration<Domain.Declaration>
     {
         public void Configure(EntityTypeBuilder<Domain.Declaration> builder)
         {
-            builder.ToTable("Declarations");
-            builder.HasKey(x => x.Id);
-            builder.Property(x => x.Id).HasColumnName(@"Id").HasColumnType("int").IsRequired();
-            builder.Property(x => x.EmpRef).HasColumnName(@"EmpRef").HasColumnType("varchar").HasMaxLength(50).IsRequired();
-            builder.Property(x => x.SubmissionTime).HasColumnName(@"SubmissionTime").HasColumnType("DateTime").IsRequired();
-            builder.Property(x => x.Data).HasColumnName(@"Data").HasColumnType("varchar(max)").IsRequired();
+            //builder.ToTable("Declarations");
```
Comment (MarieDale3) [inline src/SFA-DAS-HMRC-API-Stub/Data/Configuration/Declaration.cs]: Class could be deleted

File: src/SFA-DAS-HMRC-API-Stub/Data/Configuration/EmployerReference.cs [hunk 2]
```diff
@@ -15,7 +15,7 @@ public void Configure(EntityTypeBuilder<Domain.EmployerReference> builder)
             builder.HasKey(x => x.Id);
             builder.Property(x => x.Id).HasColumnName(@"Id").HasColumnType("int").IsRequired();
             builder.Property(x => x.EmpRef).HasColumnName(@"EmpRef").HasColumnType("varchar").HasMaxLength(50).IsRequired();
-            builder.Property(x => x.Data).HasColumnName(@"Data").HasColumnType("varchar(max)").IsRequired();
+            //builder.Property(x => x.Data).HasColumnName(@"Data").HasColumnType("varchar(max)").IsRequired();
```
Comment (MarieDale3) [inline src/SFA-DAS-HMRC-API-Stub/Data/Configuration/EmployerReference.cs]: Delete class

**Review by MarieDale3 — CHANGES_REQUESTED** [review]
Delete a couple of classes

File: src/SFA-DAS-HMRC-API-Stub/Data/Configuration/Declaration.cs (hunk 1 above)
Comment (DannyGB) [reply src/SFA-DAS-HMRC-API-Stub/Data/Configuration/Declaration.cs]: Correct, will remove

File: src/SFA-DAS-HMRC-API-Stub/Data/Configuration/EmployerReference.cs (hunk 2 above)
Comment (DannyGB) [reply src/SFA-DAS-HMRC-API-Stub/Data/Configuration/EmployerReference.cs]: Will remove

**Review by MarieDale3 — APPROVED** [review]

**Pull request merged**